goon/browserassets/css/browserOutput-dark.css

```diff
@@ -242,162 +242,161 @@ a.popt {text-decoration: none;}
 }
 
 /* OUTPUT COLORS */
-.highlight {background: yellow;}
-h1, h2, h3, h4, h5, h6	{color: #06f;font-family: Georgia, Verdana, sans-serif;}
-em						{font-style: normal;	font-weight: bold;}
-.motd					{color: #638500;	font-family: Verdana, sans-serif;}
+.highlight				 {background-color: #ffff0026;}
```

Comment:
I assume the 2 extra hex-digits are for transparency?

Reply:
They don't seem to affect anything, nor harm anything, It seems like its RGB only, no RGBA, which is frustrating.

Reply:
can we get a confirmation on this not affecting things in-game? If I remember correctly, the alpha channel for hex colors aren't supported by IE11 for background-color.

Reply:
Highlight affects CSS, all it will do is not use the values.